content/services/development.md

```diff
@@ -75,6 +75,30 @@ summary: |
 
 {% endcall %}
 
+## Design & Strategy
```

Comment:
We can try this and see. My hunch is that he's looking for specific usage of the terms "branding" and "advertising", and you've successfully removed most of those from the earlier version of the text. 😄

Reply:
Why do we need to add "advertising" if we don't offer that service?

Reply:
We could change the title to "Design & Branding" for now, if needed.

Reply:
Because the trademark classes cover the following, and I think he's looking for samples to support each one:

- Branding services, namely, consulting, development, management and marketing of brands for businesses and/or individuals
- Advertising services, namely, creating corporate logos for others
- Consulting services in the field of providing online, non-downloadable software and applications
- Internet-based application service provider, namely, hosting, managing, developing, analyzing, and maintaining the code, applications, and software for web sites of others
- Design and development of on-line computer software systems

Reply:
We do offer branding stuff, and I think that's what we intended with our trademark - so we could at least try and highlight that more.

```suggestion
## Design, Branding & Strategy
```